tests/bluetooth/controller/ctrl_ull_dletime/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+#
+# the following allows us to test the different PHY configurations
+# by setting the appropriate flags in the yaml file
+#
+if(CONFIG_BT_CTLR_PHY)
+add_definitions(-DCONFIG_BT_CTLR_PHY)
+endif(CONFIG_BT_CTLR_PHY)
+
+if(CONFIG_BT_CTLR_PHY_2M)
+add_definitions(-DCONFIG_BT_CTLR_PHY_2M)
+endif(CONFIG_BT_CTLR_PHY_2M)
+
+if(CONFIG_BT_CTLR_PHY_CODED)
+add_definitions(-DCONFIG_BT_CTLR_PHY_CODED)
+endif(CONFIG_BT_CTLR_PHY_CODED)
+
+if(CONFIG_BT_CTLR_DATA_LENGTH_MAX)
+add_definitions(-DCONFIG_BT_CTLR_DATA_LENGTH_MAX=${CONFIG_BT_CTLR_DATA_LENGTH_MAX})
+set (CMAKE_C_FLAGS "-DCONFIG_BT_CTLR_DATA_LENGTH_MAX=${CONFIG_BT_CTLR_DATA_LENGTH_MAX}")
```

Comment:
just a small note.
This will clear any existing `CMAKE_C_FLAGS`, but as it is in the main CMakeLists.txt file, it should be safe.
So this is more a general info about being careful when setting variables this way.

Reply:
agree, the set should be removed, since it  is the add_compile_definitions